charts/rancher-vsphere-csi/102.1.0+up3.0.1-rancher1/templates/node/windows-daemonset.yaml

```
@@ -57,7 +57,7 @@ spec:
             - name: ADDRESS
               value: 'unix://C:\\csi\\csi.sock'
             - name: DRIVER_REG_SOCK_PATH
-              value: {{ .Values.csiNode.prefixPath }}'\\var\\lib\\kubelet\\plugins\\csi.vsphere.vmware.com\\csi.sock'
+              value: 'C:\\var\\lib\\kubelet\\plugins\\csi.vsphere.vmware.com\\csi.sock'
```

Probably should make this something like:
```suggestion
              value: '{{ .Values.csiNode.prefixPathWindows }}\\var\\lib\\kubelet\\plugins\\csi.vsphere.vmware.com\\csi.sock'
```